<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINLT-744" failures="0" tests="4" name="com.atmecs.api.get.testscripts.TC_Get_No_Auth" time="7.146" errors="0" timestamp="04 Jun 2020 12:00:56 GMT" skipped="0">
  <testcase classname="com.atmecs.api.get.testscripts.TC_Get_No_Auth" name="getBooksCount" time="1.111"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_Get_No_Auth" name="GetInvalidRangeError" time="3.489"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_Get_No_Auth" name="getMaxResults" time="1.456"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_Get_No_Auth" name="getMissingQueryError" time="1.090"/>
</testsuite> <!-- com.atmecs.api.get.testscripts.TC_Get_No_Auth -->
